Tasks/AzureCloudPowerShellDeployment/Utility.ps1

```diff
@@ -122,7 +145,7 @@ function Get-DiagnosticsExtensions($storageAccount, $extensionsPath, $storageAcc
 
                                 Write-Verbose "##$storageAccountName = $publicConfigStorageAccountName"
                                 $storageAccountName = $publicConfigStorageAccountName
-                                $storageAccountKey = $publicConfigStorageKey.Primary
+                                $storageAccountKey = $publicConfigStorageKey
```

Comment:
I think this change is not intentional as it should be $publicConfigStorageKey.Primary